Pipfile

```diff
@@ -6,7 +6,7 @@ verify_ssl = true
 
 [requires]
 
-python_full_version = "python-3.6.4"
+python_full_version = "3.6.5"
```

Comment:
Note this is a later version of Python than Heroku supports <https://devcenter.heroku.com/articles/python-runtimes>.

IMO it's important to pin to a version that Heroku can run, for parity and also to reduce warnings from the Heroku build.

How about updating to the canonical version specifier, but (1) keeping it at 3.6.4 and (2) adding a comment that links to https://devcenter.heroku.com/articles/python-runtimes.